src/components/inputs/button/button.stories.tsx

```diff
@@ -201,3 +207,35 @@ export const IconButtons: ComponentStory<typeof Button> = () => {
     </div>
   );
 };
+
+/**
+ * File upload buttons used with or without icons.
+ */
+export const UploadButton: ComponentStory<typeof Button> = () => {
+  return (
+    <div className="story__button">
+      <input
+        accept="image/*"
+        id="icon-button-file"
+        multiple
+        type="file"
+        style={{ display: 'none' }}
+      />
+      <label htmlFor="icon-button-file">
+        <IconButton icon={<PlusCircle />} component="span" />
+      </label>
+      <input
+        accept="image/*"
```

Comment:
`Accept` should become a prop. The user should be able to pass one or more supported arguments, which are the file types the upload button should support. Check more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file) for the `accept` attribute.

Reply:
I propose the prop to expect an array of strings, which then you will pass to this prop.

Reply:
I made Accept a prop and gave it the type string[]